cloud/deploy/deploy.go

```diff
@@ -534,6 +538,11 @@ func buildImage(c *config.Context, path, currentVersion, deployImage, imageName
 				return "", err
 			}
 		} else {
+			// remove astro-run-dag from requirments.txt
```

Comment:
since this is the case for when `!dagDeployEnabled` why not call `buildImageWithoutDags()` as it is removing the line and building image too. Or else re-use the same code from a separate helper method.